drools-game-engine-services/src/main/java/org/drools/game/services/endpoint/impl/GameServiceImpl.java

```diff
@@ -79,4 +91,50 @@ public void destroy( String sessionId ) {
 		gameSession.destroy();
 	}
 
+	@Override
+	public <T> void execute(String sessionId,  CommandRest<T> cmdRest ) {
+        GameSession gameSession = games.get( sessionId );        
+        gameSession.execute( cmdRest.getCommand() );
+ 	}
+
+	@Override
+	public Player getPlayer(String sessionId, String playerName){
+		GameSession gameSession = games.get( sessionId );        
+        Player p = gameSession.getPlayerByName( playerName );
+        return p;
+	}
+	
+	@Override
+	public Zone getItem(String sessionId, String itemName){
```

Comment:
This I don't like. Game Service should have anything Game specific. So we will need a generic way to query objects. We can just expose a method to return List<Object> executeQuery(Query qa); where Query contains the name of the query + some arguments for executing the query. We can have different queries implementations in teh same way as Commands (maybe we can create a SubClass of Command to represent the queries.). This will push to the game designer to define which queries are available and how to get the arguments from the local context to be able to execute it. I will create another issue for that.